src/OpenLoco/Vehicles/VehicleBody.cpp

```diff
@@ -24,10 +24,77 @@ namespace OpenLoco::Vehicles
     static loco_global<uint8_t, 0x01136237> vehicle_var_1136237;       // var_28 related?
     static loco_global<uint8_t, 0x01136238> vehicle_var_1136238;       // var_28 related?
     static loco_global<int8_t[88], 0x004F865C> vehicle_arr_4F865C;     // var_2C related?
-    static loco_global<Map::map_pos[64], 0x00503B6A> factorXY503B6A;
     static loco_global<uint8_t[44], 0x004F8A7C> vehicle_arr_4F8A7C; // bools
     static loco_global<uint8_t, 0x00525FAE> vehicle_var_525FAE;     // boolean
 
+    // 0x00503B6A
+    // ROUND(COS((32/64+(L1/64))*(2*PI()))*256,0), ROUND(SIN(((L1/64))*(2*PI())) * 256,0)
```

Comment:
What does L1 represent here? (Presumably an Excel cell, but then what does it contain?)

I wonder if we can go one step further and generate the table at compile time using constexpr magic.